DOCUMENTATION.md

```diff
@@ -892,7 +924,29 @@ The concept that the VR user has a physical in game presence which is accomplish
    * `Transform` - The transform for the object representing the VR headset.
 
 The GetHeadset method returns the transform of the object representing the VR headset in the game world.
-  
+
+#### StartPhysicsFall/1
+
+  > `public void StartPhysicsFall(Vector3 velocity)`
+
+  * Parameters
+   * `Vector3 velocity` - The starting velocity we want to use at the start of the fall
```

Comment:
don't use `we` again. refer to `the user`

`The starting velocity the user will use at the start of the fall.`

(also remember the full stop [peroid] at the end of the line :) )